cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -3,40 +3,55 @@
 |===
 | Command | Description | Type of user | Available in Community Edition
 
-| [source, cypher, role=noplay]
+|
+[source, cypher, role=noplay]
+----
```

Comment:
Not sure about what these changes are but I guess those are updates from you?

Reply:
Yes, it is just minor fixes I did, so it is correct asciidoc :)